package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/cookie",
-  "version": "8.0.0",
+  "version": "8.0.1",
```

Comment:
revert this please, we increment on release and the one who releases sets the new version.